### PR TITLE
Update OWNERS to reflect edpm-ansible ownership

### PR DESCRIPTION
**Pull request by fao89**

### CODE REVIEW

**Review by slagle — COMMENTED** [review]
This makes several changes to the approvers/reviewers. It would  be nice to record what these actually are in the Commit msg.

Dropped from approvers:
jlarriba

Added to approvers:
fultonj
gibizer
SeanMooney

**Comment by fao89** [comment]
I actually just copied the files from edpm-ansible (I thought it was just lacking the OWNERS_ALIASES), I didn't notice it was removing/adding people

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  openstack-approvers:
+  - abays
+  - dprince
+  - olliewalsh
+  - stuggi
+  compute-approvers:
+  - gibizer
+  - SeanMooney
+  telemetry-approvers:
+  - jlarriba
```
Comment (fao89) [inline OWNERS_ALIASES:25]: added this alias for not dropping `jlarriba`
the other people/aliases came from https://github.com/openstack-k8s-operators/edpm-ansible/blob/main/OWNERS_ALIASES

**Comment by fultonj** [comment]
+2

**Review by jpodivin — APPROVED** [review]

**Pull request merged**